libraries/lib-components/PluginProvider.h

```diff
@@ -155,8 +155,6 @@ class COMPONENTS_API PluginProvider  /* not final */
       const RegistrationCallback &callback )
          = 0;
 
-   /*! @return true if the plug-in is still valid, otherwise false. */
-   virtual bool IsPluginValid(const PluginPath & path, bool bFast) = 0;
```

Comment:
And again, AudioUnitEffectsModule must update too, to keep this commit buildable